### ISSUE TITLE
integration_http_method can only be POST for Lambda

### ISSUE THREAD

**Issue by shahriar52** [theherk/terraform-aws-apigateway-proxy#3]
In the `aws_api_gateway_integration` resource definition, `integration_http_method` is derived as below,

`integration_http_method = contains(["AWS", "AWS_PROXY", "HTTP", "HTTP_PROXY"], each.value.config.type) ? aws_api_gateway_method.this[each.key].http_method : null`

That would be equal to the API Gateway resource method. For Lambda, that value can only be "POST". 

Reference: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_integration#integration_http_method

**Comment by theherk** [comment]
That's true, and I set the method as such when using lambda. If you'd prefer the module have this logic, you can suggest a change, but I'm not sure it is needed.

**Comment by theherk** [comment]
I didn't run into this because I was [setting my lambda integrations to POST already, like this](https://github.com/theherk/terraform-aws-apigateway-proxy/blob/main/examples/complete/main.tf#L31). It became apparent as soon as another verb was used. Sorry, I didn't put any further thought into it at the time.

The fix will be merged after some verification.
